tests/test_spatial_reference.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+
+import rasterio
+import pytest
+from math import pi
+
+from numpy import array
+from numpy.testing import assert_array_almost_equal
+
+from gdal2mbtiles.constants import EPSG_WEB_MERCATOR
+
+from gdal2mbtiles.gdal import SpatialReference
+
+
+# SEMI_MAJOR is a constant referring to the WGS84 Semi Major Axis.
+SEMI_MAJOR = 6378137.0
```

Comment:
@FaiBarazi, would it be worth adding these constants to `gdal2mbtiles.constants`?